src/Tickets/Commerce/Gateways/Stripe/Status.php

```diff
@@ -27,6 +27,8 @@ class Status {
 	/**
 	 * Order Status in Stripe for when the payment intent is first created or when payment is denied.
 	 *
+	 * Deprecated in favor of REQUIRES_PAYMENT_METHOD.
```

Comment:
Should we add a date/release when this was deprecated?

Reply:
Not really deprecated yet, its a Stripe thing. Once it gets moved I will actually move it.